modules/router-store/src/models.ts

```diff
@@ -0,0 +1,6 @@
+export interface RouterStateSelectors<T, V> {
+  selectQueryParams: (state: V) => string[];
```

Comment:
shouldn't this be a dictionary, since queryParams are key/value?

Reply:
I appreciate the review @yngvebn. Still tuning the PR